MiniTeks/mks-server
# Add user and developer documentation to the project

# What is Changed?
- Fixes #34
 - Add documentation for all the functions
 - Update readme
 - Cleanup

# How to test?

- Install godoc and start a go doc server

```bash
    godoc -http=:6000
```

- Open localhost:6000/pkg in yor browser and see for third-party doc
